source/ui/Image.js

```diff
@@ -174,7 +181,9 @@
 		srcChanged: function () {
 			var src = enyo.ri.selectSrc(this.src);
 			if (this.sizing) {
-				this.applyStyle('background-image', src ? 'url(' + enyo.path.rewrite(src) + ')' : 'none');
+				var placeholder =  enyo.path.rewrite(this.placeholder);
+				var multiple= "url('"+ enyo.path.rewrite(src) +"'), url('"+ (placeholder ? placeholder : enyo.Image.placeholder) + "');";
```

Comment:
I don't know if we always want a placeholder image displayed - this should be something developers opt into at the {{enyo.Image}} level, so leaving the value as the default empty string shouldn't display a placeholder.